data/brands/shop/electrical.json

```
@@ -4,6 +4,16 @@
     "exclude": {"generic": ["^electrical$"]}
   },
   "items": [
+     {
+      "displayName": "Life Saver",
+      "locationSet": {"include": ["us","ca"]},
+      "tags": {
+        "brand": "Life Saver",
+        "brand:wikidata": "Q122257514",
+        "name": "Life Saver",
```

```suggestion
      "displayName": "LifeSafer",
      "locationSet": {"include": ["us","ca"]},
      "tags": {
        "brand": "LifeSafer",
        "brand:wikidata": "Q122257514",
        "name": "LifeSafer",
```
The brand is LifeSafer, not Life Saver

---

Yup, I agree. Added the space by mistake.